src/data/migrations/20181113094807-add-missing-constraints.js

```diff
@@ -126,21 +134,21 @@ module.exports = {
 
       // return queryInterface.removeConstraint('Flows', 'userId')
```

Comment:
Looks like this whole block is not needed, right? If so, please delete.

Reply:
Removed